core/domain/state_domain_test.py

```diff
@@ -598,6 +601,133 @@ def test_validate_state_solicit_answer_details(self):
         self.assertEqual(init_state.solicit_answer_details, True)
 
 
+class ImageDomainUnitTests(test_utils.GenericTestBase):
+    """Test methods operate on Image."""
+    def test_from_and_to_dict_works_correctly(self):
```

Comment:
Add test for validation method, please make sure you are writing a test for each line you are introducing in the codebase.